packages/client/merge/README.md

```diff
@@ -11,10 +11,11 @@ So to run ethereumjs in PoS (ready) configurations, one requires to specify the
 
 Following networks have been merge to PoS with ethereumjs client having the baked in `merge` hardfork configurations:
 
-1. Ropsten (`--network ropsten`)
-2. Sepolia (`--network sepolia`)
+1. Sepolia (`--network sepolia`)
+2. Goerli (`--network goerli`)
+3. (Upcoming) Mainnet (`--network mainnet`)
```

Comment:
Do we need this one?  The client runs mainnet by default

Reply:
i felt it would be good to be explicit here to just to showcase that ethereumjs can even possibly run merged mainnet (if we can sync all the history and data, but definitely with upcoming snap sync)

Reply:
Fair enough.